includes/class-wc-gateway-dintero-hp.php

```diff
@@ -797,7 +797,14 @@ public function process_payment_handler( $order_id ) {
 					self::failed_order( $order, $dintero_order_transaction_id, $fail_reason );
 				} else {
 					$fail_reason = __( 'Failed updating transaction with order_id ' ) . '.';
-					self::failed_order( $order, $dintero_order_transaction_id, $fail_reason );
+					$order->add_order_note( $fail_reason );
+
+					$update_response_retry = WCDHP()->checkout()->update_transaction($dintero_order_transaction_id, $order_id);
+
+					if (is_wp_error($update_response_retry)) {
+						$fail_reason = __( 'Failed updating transaction with order_id. Will stop trying ' ) . '.';
```

Comment:
maybe state the effects of this error? i.e. finding the order in the settlement?

Reply:
Yup